Intersect.Editor/Forms/DockingElements/frmMapEditor.cs

```diff
@@ -179,13 +209,11 @@ public void AddUndoState()
         }
 
         //PicMap Functions
-        public void picMap_MouseDown(object sender, MouseEventArgs e)
+        private void picMap_MouseDown(object sender, MouseEventArgs e)
         {
-            if (!Globals.MapEditorWindow.DockPanel.Focused)
+            if (Globals.CurrentEditor != -1 || Globals.EditingLight != null || !picMap.Focused)
             {
-                Globals.MapEditorWindow.DockPanel.Focus();
-
-                return;
+                picMap.Focus();
```

Comment:
What is the difference between focusing the DockPanel and picMap? Is it because the cursor is only active on the map?

Reply:
yes, the cursor sprite is only active while having the cursor over the map, when moving the cursor out of it, it goes back to the cursor without a sprite - the DockPanel is kinda in between the picMap and the outside of the panel ... ? to say a thing... is what contains the picMap, and we want the cursor sprite to be drawn while having the cursor over the picMap !